### DISCUSSION TITLE
Email delivery

### DISCUSSION THREAD

**Discussion by j2is** [swellstores/community#29] [Help] [answered]
Is it possible to DKIM sign emails sent via Swell? Some customers don't receive system email notifications sent to their inbox (although being lawyers their email filtering seems to be pretty strict)

**Comment by swellmike** [comment, marked as answer]
Yes.  This is possible with Swell's SendGrid integration:

1. Enable SendGrid integration with Swell from your Admin Dashboard under `Integrations`.  You will need to manage your own SendGrid account.

2. Configure SendGrid security settings (e.g., DKIM, SPF): https://sendgrid.com/blog/how-to-use-dkim-to-prevent-domain-spoofing/
